templates/web/zurich/admin/report_edit.html

```diff
@@ -120,106 +105,131 @@
-    });
-});
-</script>
-
-[% ELSIF problem.state == 'planned' %]
+    </div>
+[% END %]
+</li>
 
-<ul class="no-bullets">
 [% list = FOR body IN bodies %]
     [%- NEXT UNLESS body.parent.id == c.user.from_body.id %]
-    <option value="[% body.id %]">[% body.name %]</option>
+    <option value="[% body.id %]"[% IF body.id == problem.bodies_str %] selected[% END %]>[% body.name %]</option>
 [% END %]
   [% IF admin_type != 'super' AND list %]
-    <li class="assignation">
-    <label for="body_subdivision">[% loc('Assign to subdivision:') %]</label>
+    <li class="assignation" id="assignation__subdivision">
+        <label for="body_subdivision">[% loc('Assign to subdivision:') %]</label>
         <select name="body_subdivision" id="body_subdivision">
-            <option value="" selected>--</option>
+            <option value="">--</option>
             [% list %]
         </select>
     </li>
   [% END %]
 
-    <li><label for="status_update">[% loc('Public response:') %]</label>
-    [% INCLUDE 'admin/response_templates_select.html' for='status_update' %]
-    <textarea name='status_update' id='status_update' cols=60 rows=5>
-      [%- IF problem.extra.public_response -%]
-        [%- problem.extra.public_response | html -%]
-      [%- ELSE -%]
-
-Freundliche Grüsse
+[% END %]
 
-Ihre Stadt Zürich
-    [%- END %]</textarea>
+[%# 3rd party messages sent for Extern/Wunsch states %]
+[% FOR s IN ['closed','investigating'] %]
+    [% IF problem.state == s %]
+    <ul class="no-bullets">
+    <li class="assignation" id="assignation__external">
+    <label for="body_external">
+        [% IF problem.state == 'closed' %]
+            [% loc('Assign to external body:') %]
+        [% ELSE %]
+            [% loc('Assign to competent body:') %]
+        [% END %]
+    </label>
+        <select name="body_external" id="body_external">
+            <option value="">--</option>
+        [% FOR body IN bodies %]
+            [% NEXT IF body.parent OR body.bodies OR body.deleted %]
+            <option value="[% body.id %]"[% IF body.id == problem.external_body %] selected[% END %]>[% body.name %]</option>
+        [% END %]
+        </select>
+        <br>
+        [% IF problem.state == 'closed' %]
+        <input type="checkbox" name="third_personal" id="third_personal" value="1"[% ' checked' IF problem.extra.third_personal %]>
+        <label for="third_personal" class="inline">[% loc('Include reporter personal details') %]</label>
+        [% END %]
+        [% INCLUDE 'admin/response_templates_select.html' for='external_body' %]
+        <textarea name='external_message' id='external_body' cols=60 rows=5></textarea>
     </li>
-</ul>
+    </ul>
+    [% END %]
+[% END %]
 
-[% ELSIF problem.state == 'fixed - council' %]
+[%# Public response field shown for Ruckmeldung ausstehend/Extern/
+  # Zustandigkeit unbekannt/Wunsch/Nicht kontaktierbar/Unsichtbar %]
+[% FOR s IN ['planned','closed','unable to fix','investigating','partial','hidden'] %]
```

Comment:
Similarly to above, but with the increased number of possible states this would be better as a `SWITCH`.